ci/requirements-py27-windows.yml

```diff
@@ -21,4 +20,4 @@ dependencies:
   - rasterio
   - zarr
   - pip:
-    - cftime
+    - netcdf4
```

Comment:
Latest conda-forge netcdf4 does use cftime.

Reply:
Thanks @ocefpaf -- we need to use pip in this case only since we use this environment to test on the win-32 platform: https://github.com/conda-forge/cftime-feedstock/issues/2.